auth-api/src/auth_api/__init__.py

```diff
@@ -74,7 +74,7 @@ def execute_migrations(app):
         upgrade(directory="migrations", revision="head", sql=False, tag=None)
     except Exception as e:  # NOQA pylint: disable=broad-except
         app.logger.disabled = False
-        app.logger.error("Error processing migrations:", exc_info=True)
+        app.logger.error("Error processing migrations: %", str(e))
```

Comment:
hopefully the error isn't complicated, this wont be sufficient

Reply:
![image](https://github.com/user-attachments/assets/c110e825-bcad-45ba-8219-cb72ef692969)